### DISCUSSION TITLE
A way to remove the language indicator on the taskbar?

### DISCUSSION THREAD

**Discussion by Volern** [valinet/ExplorerPatcher#2032] [Q&A] [answered]
Super annoying have it there. I turn it off in the notification icon settings opened through the properties window, but explorer patcher will restart file explorer after a short while and re-enable it.  Anything I can do to permanently remove it?
![image](https://github.com/valinet/ExplorerPatcher/assets/13373330/cbb23291-ebd0-4ca1-9a9d-e8abf0bf3ade)

**Comment by tallman69** [comment]
This may help.  #1753

**Reply by Volern** [reply, marked as answer]
What he does doesn't work for me as EP restarts but it did inform me that having 2 languages makes it show up so I deleted the English US and kept my English UK and it went away.
